<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerViewItems</class>
 <widget class="QWidget" name="CustomerViewItems">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CustomerViewItems</string>
  </property>
  <widget class="QTextEdit" name="ViewByName_name">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>370</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;View By Category&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Category&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ViewByCategory_list">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;View By Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ViewByNameNext">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ViewByCategoryNext">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>-30</y>
     <width>511</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#00aaff;&quot;&gt;Hello Customer, You can view all the items here.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>530</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>BackButton</sender>
   <signal>clicked()</signal>
   <receiver>CustomerViewItems</receiver>
   <slot>BackButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>720</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ViewByNameNext</sender>
   <signal>clicked()</signal>
   <receiver>CustomerViewItems</receiver>
   <slot>ViewByName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ViewByCategoryNext</sender>
   <signal>clicked()</signal>
   <receiver>CustomerViewItems</receiver>
   <slot>ViewByCategory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>591</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BackButton()</slot>
  <slot>ViewByName()</slot>
  <slot>ViewById()</slot>
  <slot>ViewByCategory()</slot>
 </slots>
</ui>
